modules/credentials/outputs.tf

```
@@ -1,3 +1,4 @@
 output "metric_writer_credentials_file" {
-  value = google_service_account_key.metric_writer.private_key
+  value       = google_service_account_key.metric_writer.private_key
+  description = "The private key of the Metrics Writer Service Account."
```

I think this is how google themselves name these 

```suggestion
  description = "The credential file of the Metrics Writer Service Account."
```